pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -175,48 +174,25 @@ func TestGetPipelineFunc(t *testing.T) {
 		name            string
 		localPipelines  []runtime.Object
 		remotePipelines []runtime.Object
-		ref             *v1beta1.PipelineRef
+		ref             *v1.PipelineRef
 		expected        runtime.Object
 	}{{
-		name: "remote-pipeline",
```

Comment:
I'd prefer to keep these test cases around, similar to https://github.com/tektoncd/pipeline/pull/6778

Reply:
I opened https://github.com/tektoncd/pipeline/pull/6787/commits if that could help to clarify the changes here.

I think the remote-pipeline here could be removed since it is related with bundle syntax and similar to https://github.com/tektoncd/pipeline/pull/6444#discussion_r1224463886.

And the `remote-pipeline-without-default` case could be covered by ["v1 pipeline] (https://github.com/tektoncd/pipeline/blob/647a5cefd4371719e207ce110bd8238a728841ce/pkg/reconciler/pipelinerun/resources/pipelineref_test.go#L345)